main_app/templates/main_app/women_laws.html

```diff
@@ -12,8 +12,6 @@
             background-color: black;
```

Comment:
The fonts for the nav should be as previous. Undo the changes that are done.
![image](https://user-images.githubusercontent.com/53635792/112321066-9de6bf80-8cd5-11eb-9ac3-dc58ba23c38a.png)